crates/ruff_linter/src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -170,6 +170,9 @@ impl AlwaysFixableViolation for PytestExtraneousScopeFunction {
     }
 }
 
+/// ## Deprecation
+/// Marking fixtures that do not return a value with an underscore isn't a practice recommended by the pytest community.
+///
```

Comment:
If someone more familiar with pytest has a better explanation, please comment.

Reply:
I prefer to keep the description as is. By convention doesn't mean that it has to be a blessed convention ;) It can be your personal convention. I also don't see the point of improving the documentation of a deprecated rule.

Reply:
> I prefer to keep the description as is. By convention doesn't mean that it has to be a blessed convention ;) It can be your personal convention. I also don't see the point of improving the documentation of a deprecated rule.

While that's strictly-speaking true, that seems like quite a legalistic interpretation to me :P that's not how I understood the phrase in the docs, and I doubt it's how many of our users would understand it either.

> I also don't see the point of improving the documentation of a deprecated rule.

Hmm, that logic seems circular to me. Users might not know that it's deprecated (or why it's deprecated) until they read the documentation that tells them about the rule. I think documentation is still really important here so that we can clearly convey why we no longer endorse the rule's recommendations